xml/release-notes.xml

```diff
@@ -664,12 +664,14 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_VALUE</replaceable>
    <itemizedlist>
     <listitem>
      <para>
-      GNOME will set DPI according to each monitor's reported scaling.
+      GNOME will set <literal>Xft.dpi</literal> to the configured multiple of
+      96 DPI for each monitor.
      </para>
     </listitem>
     <listitem>
      <para>
-      KDE Plasma disables automatic scaling by default.
+      KDE Plasma disables the automatic scaling of Qt but uses an own scaling
```

Comment:
It configues Qt manually, so does not rely on DPI autodetection. The scaling mechanism is still the same in Qt.

Reply:
Thanks for all the clarifications! It should now be correct.